pkg/repo/migrations/v3_4_test.go

```diff
@@ -40,7 +40,8 @@ func TestV3MigrationsTestSuite(t *testing.T) {
 	suite.Run(t, new(V3MigrationsTestSuite))
 }
 
-func (suite *V3MigrationsTestSuite) TestV3MigrationWithDefaultRepo() {
```

Comment:
I've increased test coverage for a variety of possible repos:
- repo generated from `bacalhau serve --node-type=requester,compute`
- repo generated from `bacalhau serve --node-type=requester`
- repo generated from `bacalhau serve --node-type=compute`
-  repo generated from `bacalhau version`